paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -1170,7 +1177,6 @@ void CPUQuantizePass::ApplyImpl(ir::Graph* graph) const {
   QuantizeImmutable(graph, "reshape2", "X");
   QuantizeImmutable(graph, "transpose2", "X");
   QuantizeImmutable(graph, "slice", "Input");
```

Comment:
If you added a check above, it seems to me that this shape deletion is not needed

Reply:
shape OP will never be quantized, so I think this line of code is redundant.